force-app/main/default/classes/Integration Recipes/ConnectApiWrapper.cls

```diff
@@ -0,0 +1,23 @@
+public with sharing class ConnectApiWrapper {
```

Comment:
I would add a one liner description for this class explaining why we created it since this seems to be reusable to wrap any ConnectApi related functions in future. 

Also I would not place this here and instead move to the Shared Code folder so we reuse it.

Reply:
Good point.